src/centralserver/admin-ui/src/views/InitialConfiguration/InitialConfiguration.vue

```diff
@@ -334,6 +337,13 @@ export default (
           400 === errorStatus && 'validation_failure' === error?.error?.code
         );
       }
+
+      function isWeakPinError(error: ErrorInfo) {
+        let errorStatus = error.status;
```

Comment:
The `let` keyword should be replaced with `const` since the variable is not mutated at any point. Actually all `let` variable declarations in this file should probably be replaced with `const`

Reply:
Fixed in https://github.com/nordic-institute/X-Road/pull/1101/commits/4872b6c6dcbc4b80f571e61fa70dfd59c0a67694